src/kayenta/actions/index.ts

```diff
@@ -45,6 +45,7 @@ export const UPDATE_SCORE_THRESHOLDS = 'update_score_thresholds';
 export const UPDATE_GROUP_WEIGHT = 'update_group_weight';
 export const SET_CONFIG_JSON_MODAL_TAB_STATE = 'set_config_json_modal_tab_state';
 export const UPDATE_ATLAS_QUERY = 'update-atlas-query';
+export const UPDATE_DATADOG_METRIC_TYPE = 'update-datadog-metric-type';
```

Comment:
It looks like the delimiter here has been inconsistent - could you please switch this one and the others to use an underscore?

Reply:
Good catch, updated